tests/pytests/common.py

```diff
@@ -380,6 +380,8 @@ def wrapper():
                 raise SkipTest()
             if gc_no_fork and Env().cmd(config_cmd(), 'GET', 'GC_POLICY')[0][1] != 'fork':
                 raise SkipTest()
+            if NOJSON and not REJSON:
```

Comment:
to support both `True` and `False` we should have it like we do with `cluster`:
```suggestion
            if NOJSON == REJSON:
```